src/js/actions/guides.js

```diff
@@ -270,15 +335,17 @@ define(function (require, exports) {
 
             if (target && _.isArray(target) && target.length === 2 &&
                 target[0]._ref === "document" && target[1]._ref === "good") {
-                var payload = {
-                    documentID: target[0]._id,
-                    layerID: event.layerID,
-                    index: target[1]._index - 1, // PS indices guides starting at 1
-                    orientation: event.orientation._value,
-                    position: event.position._value
-                };
-
-                this.flux.actions.guides.setGuide(payload);
+                var mockDocument = {
+                        id: target[0]._id
+                    },
+                    mockGuide = {
```

Comment:
Also weird? Seems like the canonical parameter list ought to be: `Document, Layer, orientation, position, index`.